daemon/kmd/wallet/driver/ledger.go

```diff
@@ -355,7 +355,7 @@ func (lw *LedgerWallet) SignProgram(data []byte, src crypto.Digest, pw []byte) (
 }
 
 // MultisigSignTransaction implements the Wallet interface.
-func (lw *LedgerWallet) MultisigSignTransaction(tx transactions.Transaction, pk crypto.PublicKey, partial crypto.MultisigSig, pw []byte) (crypto.MultisigSig, error) {
+func (lw *LedgerWallet) MultisigSignTransaction(tx transactions.Transaction, pk crypto.PublicKey, partial crypto.MultisigSig, pw []byte, signer crypto.Digest) (crypto.MultisigSig, error) {
```

Comment:
I'm abit confused. It seems that you've done a lot in order to pass on the signer to this function, but when you reached this function it doesn't look like you're doing anything. What am I missing ?

Reply:
Yes, I basically had to add it because of the interface even though it doesn't do much. @algomaxj